Cargo.toml

```diff
@@ -25,7 +23,6 @@ members = [
     "common/short-hex-str",
     "common/temppath",
     "common/time-service",
-    "common/workspace-hack",
```

Comment:
I really wish we had a better name for this than `workspace-hack`

Reply:
We'll we didn't' really name it, its something that was named by the rust lang project